garden-backend-service/src/api/routes/doi.py

```diff
@@ -32,9 +38,44 @@ async def mint_draft_doi(
     return {"doi": result.data.attributes.doi}
 
 
-@router.put("/", status_code=status.HTTP_200_OK)
+@router.put("", status_code=status.HTTP_200_OK)
 async def update_datacite(
-    body: datacite.Doi,
+    body: datacite.Doi = Body(
+        examples=[
+            {
+                "data": {
+                    "type": "dois",
+                    "attributes": {
+                        "types": {
+                            "resourceType": "AI/ML Garden",
+                            "resourceTypeGeneral": "Software",
+                        },
+                        "identifiers": [
+                            {
+                                "identifier": "10.23677/fakedoi-1234",
+                                "identifierType": "DOI",
+                            }
+                        ],
+                        "creators": [
+                            {
+                                "name": "Mendel, Gregor",
+                            }
+                        ],
+                        "titles": [
+                            {
+                                "title": "GPT Garden (Generative Pea-trained Transformers)",
```

Comment:
usually hate to revive a dead joke but I really needed to get this out of my system